bionlp/taggers/rnn_feature/tagger.py

```diff
@@ -71,7 +75,16 @@ def train_NN(train, crf_output, lstm_output, train_indices, compute_cost, comput
                 if patience_has_ended:
                     sl.info("Stopping because my patience has reached its limit.")
                     break
-            
+            if params['model'] is not 'None' and params['save-interval'] is not 0:
+                save_interval = params['save-interval']
```

Comment:
theres no need to assign an extra variable here, `save_inerval` is only used once